ynput/ayon-core
# Nuke: Support options for reading deep exr in load clips

## Changelog Description
This PR is to add the support of reading deep exr in load clips. Users can choose to turn on/off of reading the deep exr node. By default is False.

## Additional info
Make sure you have built the latest nuke addon for this support.

## Testing notes:
1. Go to `ayon+settings://nuke/load/LoadClip/options_defaults/deep_exr` to turn on the toggle
2. Or you can choose the options through  `AYON->Load...` after launching Nuke
![image](https://github.com/ynput/ayon-core/assets/64118225/9ef8ab89-1e50-4d64-9bfb-5d62f0f89e92)
![image](https://github.com/ynput/ayon-core/assets/64118225/fae973aa-c128-462d-a2f4-fae97c063310)
3. Load the clips
4. The deep read node should be used for loading clip while the deep exr option turns on. The read node should be used when the option turns off.
